Move Katie Mazaitis to past members (#850)
cmu-delphi/www-main: content/people/index.md

  image: kathryn-mazaitis-500x500-min.jpg
  affiliation: CMU/MLD
  team:
  - past
  - blog
  note: Core member, March 2020-July 2023
  link: https://cs.cmu.edu/~krivard

>   description: leads Delphi's engineering team, and is a Principal Research Programmer in the Machine Learning Department at CMU.

Reviewer: maybe update this to past tense?  ¯\\\_(ツ)_/¯ 
```suggestion
  description: previously led Delphi's engineering team, and was a Principal Research Programmer in the Machine Learning Department at CMU.
```
